build.zig.zon does not have test path added

For me using zig 0.13.0, when zig-yaml is fetched it fails to build because the test directory is missing.

❯ zig build run
/home/gota/.cache/zig/p/1220475e37c9612bdbf77903454e9fc827b0c008a2db3d82a78387b0c55cc736fc5f/build.zig:2:26: error: unable to load '/home/gota/.cache/zig/p/1220475e37c9612bdbf77903454e9fc827b0c008a2db3d82a78387b0c55cc736fc5f/test/spec.zig': FileNotFound
const SpecTest = @import("test/spec.zig");

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -3,6 +3,7 @@
     .version = "0.0.1",
     .paths = .{
         "src",
+        "test",
         "build.zig",
         "build.zig.zon",
         "LICENSE",

